brianlong/www.validators.app
# 183794524 - decrease leader slot timeout

#### What's this PR do?
PR adds separated leader scripts for mainnet and testnet. Also, the sleep time within the tasks has been decreased to 0.5 sec.

#### How should this be manually tested?
- run the `rails r daemons/leader_stats_mainnet_update_daemon.rb` and `rails r daemons/leader_stats_testnet_update_daemon.rb` and check if the leader component in the homepage displays correctly.
- testing STAGING server: make sure the stale leaders daemon task has been stopped. You can run the following via ssh:
`systemctl --user stop leader_stats_update_daemon.service`

#### What are the relevant tickets?
[- URL to the PivotalTracker ticket](https://www.pivotaltracker.com/story/show/183794524)

#### Task completed checklist:
- [] Is there appropriate test coverage?
- [] Did I take into consideration possible security vulnerabilities?
- [] Are the controllers secure?
- [] Was every important matter documented?

#### Things to watch out for
- [N] Does this PR have any migrations?
- [N] Does this PR add new dependencies?
